tools/docker/scripts/install_coverage.sh

```diff
@@ -7,8 +7,8 @@
 # Hence, we're using https://github.com/linux-test-project/lcov/commit/75fbae1cf
 # For details see https://github.com/linux-test-project/lcov/issues/58
 
-lcov_version="75fbae1cfc5027f818a0bb865bf6f96fab3202da"
-lcov_sha256="f536c31d517d54b36b8a15fceb9e697c05f920bb31ac6fb1a43d19eafa077fa6"
+lcov_version="1.15"
```

Comment:
Thanks to this upgrade the comment above will become obsolete.

Reply:
thanks, done